x-pack/plugins/siem/package.json

```diff
@@ -1,7 +1,9 @@
 {
   "author": "Elastic",
   "name": "siem",
-  "version": "7.0.0-alpha1",
+  "version": "8.0.0",
```

Comment:
++ Thanks for the update! It should `8.0.0` solid correct, no pre-release moniker?

Reply:
That is the same version number that is stamped on the main `package.json` level of kibana at the moment:
https://github.com/elastic/kibana/blob/master/package.json#L14

So that is why I made that change. It looks good to me! :-)